activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -1,4 +1,7 @@
 module ActiveSupport
+  # These load hooks are how Rails communicates that a component of Rails like
```

Comment:
I really fell that this is explained in the next paragraph. We even have an example on how that works for `ActiveRecord::Base`.